### PR TITLE
Possible patches, refactoring and maybe cleanup.

### PR DESCRIPTION
**Pull request by saas786**

### CODE REVIEW

**Comment by kylebrodeur** [comment]
Task linked: [CU-2xw2h8u Aybolit review / testing / patches after playbooks -> master merge](https://app.clickup.com/t/2xw2h8u)

**Comment by anoblet** [comment]
What happened to https://github.com/conversionxl/aybolit/pull/198?

I picked up that error a couple of weeks ago: https://github.com/conversionxl/aybolit/pull/202#discussion_r841630764.

It looks like `CXLPlaybookAccordion` supports a single object argument (https://github.com/conversionxl/aybolit/blob/476764d8fcb147e69b7324563e3f0d64153b476c/packages/storybook/cxl-ui/cxl-vaadin-accordion/cxl-playbook-accordion.story.js#L5)

Maybe use this instead?:

```js
${CXLPlaybookAccordion({ FeedbackButtonLabel: 'Leave feedback', PlaybookId: '1234' })}
```

File: packages/storybook/cxl-ui/cxl-paywall/cxl-paywall-layout=2c-l.stories.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ const Template = ({ _count, _limit, delay, opacity, subscribed }) => html`
         opacity=${opacity}
         ?subscribed=${subscribed}
       >
-        <footer class="entry-footer">${CXLPlaybookAccordion()}</footer>
+        <footer class="entry-footer">${CXLPlaybookAccordion('Leave feedback', '1234')}</footer>
```
Comment (anoblet) [inline packages/storybook/cxl-ui/cxl-paywall/cxl-paywall-layout=2c-l.stories.js:131]:
```suggestion
        <footer class="entry-footer">${CXLPlaybookAccordion({ FeedbackButtonLabel: 'Leave feedback', PlaybookId: '1234' })}</footer>
```

**Review by anoblet — CHANGES_REQUESTED** [review]

**Comment by saas786** [comment]
> What happened to the story from https://github.com/conversionxl/aybolit/pull/198?

This has been moved to new general story: https://github.com/conversionxl/aybolit/commit/675f29b71d87c7ee70a2c129b5c7e28524c1d25d

> I picked up on the error a couple of weeks ago: https://github.com/conversionxl/aybolit/pull/202#discussion_r841630764.
> It looks like `CXLPlaybookAccordion` supports a named object argument. Using it would help future-proof things. 

Thanks, I missed it, too many components to look for, PRs etc. Updated the code, should be fine now.

**Review by anoblet — APPROVED** [review]
LGTM

**Pull request merged**